core/src/main/java/org/apache/carbondata/core/preagg/TimeSeriesUDF.java

```diff
@@ -76,6 +79,25 @@ public Timestamp applyUDF(Timestamp data, String function) {
         calendar.set(Calendar.MILLISECOND, 0);
         calendar.set(Calendar.SECOND, 0);
         break;
+      case FIVE_MINUTE:
+        getData(calendar, 5);
```

Comment:
it is a set function right ? rename to setData

Reply:
you are right, done